doc/source/ray-overview/installation.rst

```diff
@@ -272,7 +272,7 @@ You can install and use Ray C++ API as follows.
 M1 Mac (Apple Silicon) Support
 ------------------------------
 
-Ray has experimental support for machines running Apple Silicon (such as M1 macs).
+Ray supports machines running Apple Silicon (such as M1 macs).
 Multi-node clusters are untested. To get started with local Ray development:
 
 #. Install `miniforge <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh>`_.
```

Comment:
Actually do you know if these miniforge instructions are still needed? Was this primarily for installing `grpcio`?

Reply:
It is just a convenient way to get things up and running, that's why I kept it. When I was trying stuff out, these instructions helped to get a good Python distribution installed, that's why I put it back. But no strong opinions here :)

Reply:
Ah I see. Reading over this section again now, I feel that there shouldn't be any more unique differences from the other operating systems which are covered by the [Official Releases](https://docs.ray.io/en/master/ray-overview/installation.html#official-releases) section.

Any thoughts on recommending using conda across different operating systems, rather than just for M1?

Reply:
In general, we probably shouldn't be in the business of recommending Python distributions to people. On M1 there is a little more ambiguity since e.g. you could use an x86 distribution and Rosetta (which grpc actually recommended for a while) and some of the non-conda distributions might be less well supported.

I'll merge the PR now because it strictly improves things, but happy to remove this section altogether in a follow up PR :)